### DISCUSSION TITLE
Photos not showing in Swell storefront

### DISCUSSION THREAD

**Discussion by PairingsCigarBar** [swellstores/community#261] [Help] [answered]
I had a developer create our storefront
https://pairingscigarbar.com/collection/cigar
Had no problem creating items originally, but the last two, I cannot get the images to show in the storefront. I merely cloned other items, so i can't figure out why photos will not show for the last few products on the storefront page.

**Comment by sags95** [comment, marked as answer]
I believe in this case your developer built the site using Next.js and has only added the `cdn.schema.io` domain to the `next.config` file. They will also need to add `cdn.swell.store` domain to that file since it looks like those images belong to that domain.
